Revised introduction (#13)
greenelab/word_lapse_manuscript: content/02.introduction.md

Studying these shifts can provide a nuanced understanding of how language adapts to describe our world.

Scientific fields of inquiry also change, sometimes rapidly, as researchers devise and test new hypotheses and applications.
For example, repurposing the CRISPR-Cas9 system to a pervasive tool for genome editing has altered how we discuss molecular entities.
Microbes use this as an immune system to defend against viruses.
Scientists repurposed this system for genome editing [@doi:10.1126/science.1225829], leading to changes in the use of the term.
Science is a field with substantial written communication [@doi:10.1021/ci00050a001], both via published papers [@doi:10.1073/pnas.98.2.381] and preprints [@doi:10.1101/833400; @doi:10.1126/science.aay2933].
Examining scientific manuscripts with computational linguistics can reveal longitudinal trends in scientific research.

The task of examining if words change their meanings is called semantic shift detection.
This process involves capturing word usage patterns such as frequency and structure over a set period of time [@arxiv:1806.03537].

> Once these patterns have been captured, the last step is utilize these patterns to generate a time series that will show possible shift events, commonly referred to as changepoints [@arxiv:1806.03537;@arxiv:0710.3742,@gustafsson].

Reviewer: ```suggestion
Once these patterns have been captured, the last step is generating a time series showing possible shift events, commonly called changepoints [@arxiv:1806.03537;@arxiv:0710.3742,@gustafsson].
```

Scientists have used this approach to discover many changepoints within publically available English corpora [@doi:10.1145/2736277.2741627; @doi:10.1109/JCDL.2014.6970173; @doi:10.1145/2064448.2064475; @doi:10.18653/v1/N18-1044; @doi:10.1017/S1351324918000220].

> These discoveries consisted of semantic changes such as awful moving from the meaning of majestic to the meaning of horrible [@arxiv:1605.09096].

Reviewer: ```suggestion
These discoveries included semantic changes like the meaning of awful shifting from majestic to horrible [@arxiv:1605.09096].
```
